ddtrace/tracer/civisibility_transport.go

```diff
@@ -0,0 +1,201 @@
+// Unless explicitly stated otherwise all files in this repository are licensed
+// under the Apache License Version 2.0.
+// This product includes software developed at Datadog (https://www.datadoghq.com/).
+// Copyright 2024 Datadog, Inc.
+
+package tracer
+
+import (
+	"bytes"
+	"compress/gzip"
+	"fmt"
+	"io"
+	"net/http"
+	"os"
+	"runtime"
+	"strconv"
+	"strings"
+
+	"gopkg.in/DataDog/dd-trace-go.v1/internal"
+	"gopkg.in/DataDog/dd-trace-go.v1/internal/civisibility/constants"
+	"gopkg.in/DataDog/dd-trace-go.v1/internal/log"
+	"gopkg.in/DataDog/dd-trace-go.v1/internal/version"
+)
+
+// Constants for CI Visibility API paths and subdomains.
+const (
+	TestCycleSubdomain = "citestcycle-intake" // Subdomain for test cycle intake.
+	TestCyclePath      = "api/v2/citestcycle" // API path for test cycle.
+	EvpProxyPath       = "evp_proxy/v2"       // Path for EVP proxy.
+)
+
+// Ensure that civisibilityTransport implements the transport interface.
+var _ transport = (*civisibilityTransport)(nil)
+
+// civisibilityTransport is a structure that handles sending CI Visibility payloads
+// to the Datadog endpoint, either in agentless mode or through the EVP proxy.
+type civisibilityTransport struct {
```

Comment:
Same as previously:

```suggestion
type ciVisibilityTransport struct {
```

Reply:
fixed in [465456a](https://github.com/DataDog/dd-trace-go/pull/2739/commits/465456aeed42aa1b9a6b4a96c0dad5a402e09308)